testing/tiptop/APKBUILD

```diff
@@ -0,0 +1,28 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=tiptop
+pkgver=2.3
+pkgrel=0
+pkgdesc="Tiptop: Hardware Performance Counters for the Masses"
+url="http://tiptop.gforge.inria.fr"
+arch="x86_64"
```

Comment:
Thanks for the insight! `x86` and `x86_64` is fine then, together with a comment saying that the appropriate perf counters are required. We can always expand them later.